view/theme/frio/templates/filebrowser.tpl

```diff
@@ -8,9 +8,11 @@
 		</div>
 
 		{{* The breadcrumb navigation *}}
-		<ol class="path breadcrumb" aria-label="Breadcrumb" role="navigation">
+		<ol class="path breadcrumb" aria-label="Breadcrumb" role="menu">
```

Comment:
Why have you changed this?

Reply:
`navigation` isn't a valid value for the `role` attribute.

Reply:
It would be best if @pvagner was on the develop branch. I guess here neither of us can really judge about the changes :-)